addons/website/static/src/snippets/s_process_steps/options.js

```diff
@@ -146,8 +147,10 @@ options.registry.StepsConnector = options.Class.extend({
      * @returns {integer}
      */
     _getStepColPadding(stepEl) {
-        const paddingClass = stepEl.className.split(' ').find(cl => cl.startsWith('offset-lg'));
-        return paddingClass ? parseInt(paddingClass[paddingClass.length - 1]) : 0;
+        const classPrefix = 'offset-lg';
```

Comment:
Shouldn't there be an hyphen at the end of this string ? Otherwise the parsed int will become a negative number ?

Reply:
Ho! good catch :open_mouth:  It was a distraction error

It's now fixed :+1:

Reply:
I now realise this specific behavior also happens in 14.0 and 15.0... maybe we should create a distinct PR on 14.0 for that.